sailthru/Sailthru_Client_Exception.php

```diff
@@ -1,5 +1,19 @@
 <?php
 
+/**
+ * Handles Sailthru Client Exceptions
+ */
 class Sailthru_Client_Exception extends Exception {
+    /**
+     * Sailthru internal error codes.
+     */
+    const ERROR_CODE_INTERNAL_EXCEPTION = 9;
+    const ERROR_CODE_INVALID_EMAIL = 11;
 
-}
\ No newline at end of file
+    /**
+     * Standardized exception codes.
+     */
+    const CODE_GENERAL = 1000;
+    const CODE_RESPONSE_EMPTY = 1001;
+    const CODE_RESPONSE_INVALID = 1002;
```

Comment:
Sailthru currently uses error codes from 1 to 99. Starting at 1000 allows room for growth for custom error codes and avoids collision with any new error codes Sailthru may add to the API. I'm open to starting at a higher number too if you think it's a good idea.

Reference: https://getstarted.sailthru.com/new-for-developers-overview/api/api-response-errors/